packages/trader/src/Modules/CFD/Containers/cfd-dashboard.jsx

```diff
@@ -66,7 +66,6 @@ const CFDDashboard = props => {
 
     // eslint-disable-next-line react-hooks/exhaustive-deps
     React.useEffect(() => {
-        updateActiveIndex();
```

Comment:
we should remove it, because it will be always go to condition **if (index === undefined) return;** and never will handle.  also we have another useEffect, where we have this function    React.useEffect(() => {
        **updateActiveIndex(getIndexToSet());**